tiledb/sm/storage_manager/storage_manager.h

```diff
@@ -160,22 +160,23 @@ class StorageManager {
    *
    * @param array_uri The array URI.
    * @param enc_key The encryption key to use.
-   * @param array_schema The array schema to be retrieved after the
-   *     array is opened.
-   * @param fragment_metadata The fragment metadata to be retrieved
-   *     after the array is opened.
```

Comment:
These out-parameter documentation pieces should be reworked for the `@return` item below. It's a deficiency that the multiple-return value scheme with structured binding does not allow named tuple members. All the more reason to make sure the `@return` text is clear.